tests/dummy/app/components/each-in.js

```diff
@@ -3,7 +3,7 @@ import Ember from 'ember';
 const { keys } = Object;
 const { computed, get } = Ember;
 
-export default Ember.Component.extend({
+const EachInComponent = Ember.Component.extend({
   positionalParams: ['object'],
```

Comment:
yeah, but if we reopen the class anyway and add it there then why add it up here too?